### DISCUSSION TITLE
Chrome security warning on OSX

### DISCUSSION THREAD

**Discussion by danawoodman** [pocketbase/pocketbase#509] [Q&A]
On my Macbook Air M2 on Monterrey, getting this error downloading the ARM zip:

![image](https://user-images.githubusercontent.com/157695/190790730-7be504af-1668-4622-b4a5-6fcc30c8a679.png)

Never seen this error before

**Comment by spankyed** [comment]
Looks like this can be fixed by filing a report @ https://support.google.com/webmasters/answer/9044101.

**Comment by ganigeorgiev** [comment]
I guess this warning is shown only on a mac, but I don't have access to one to verify it. This maybe is similar to issue https://github.com/pocketbase/pocketbase/issues/146.

I'll see if I can report it as false positive to google, but in the meantime if you are concerned about the zip, you can:

1. Build the base PocketBase application on your own
     Clone the repo -> go to `examples/base` and run `go build` and that's it.

2. Or go to the [releases page](https://github.com/pocketbase/pocketbase/releases) and manually check the checksums.
      All releases are generated using [github-action](https://github.com/pocketbase/pocketbase/blob/master/.github/workflows/release.yaml) with the following [goreleaser config](https://github.com/pocketbase/pocketbase/blob/master/.goreleaser.yaml).

_I'll move the issue to Q&A discussion._

**Reply by jonafeucht** [reply]
I'd like to add that Safari, Firefox, and Edge don't show this warning.

There is also nothing stopping you from clicking on keep.
<img width="514" alt="Screenshot 2022-09-17 at 08 17 49" src="https://user-images.githubusercontent.com/14951074/190843475-ff5f48b8-e167-4176-b66c-790d62efd4e9.png">
Or just using the terminal 
`curl -OL https://github.com/pocketbase/pocketbase/releases/download/v0.7.3/pocketbase_0.7.3_darwin_amd64.zip`

Both this and #146 are in my opinion very minor issues that we devs can simply ignore, the end-user isn't going to be affected by them.

**Reply by danawoodman** [reply]
You can work around them, but they can also give the impression of an unfinished or potentially untrustworthy application. I know that's not the case with PB, but I wouldn't just shrug it off as not important either.

**Reply by ganigeorgiev** [reply]
I've sent a false positive report to the VirusTotal provider that resulted in marking the release as "dangerous".

Currently I don't have access to a mac and I'm not able to sign the binary, so developers will have to ignore the warning or build the application on their own.

**Reply by mrdcmiller** [reply]
You can find a way around the warning this way.

Open the folder in Finder, and click on the executable with Control held down. 

You'll get a warning, accept it and choose open, and OSX will remember that you trust this app when you run ./pocketbase serve
